profiles/databricks.yaml

```diff
@@ -6,22 +6,31 @@
 # Set the registry server URI. This property is especially useful if you have a registry
 # server that’s different from the tracking server.
 # Profile could be created using https://github.com/databricks/databricks-cli#installation
-# model_registry:
-#   uri: "databricks://PROFILE_NAME"
+model_registry:
+  # registry_uri: "databricks://PROFILE_NAME"
+  # FIXME::REQUIRED: Specifies the name of the Registered Model to use when registering a trained
+  #                 model to the MLflow Model Registry.
+  model_name: "taxi_fare_regressor"
+  # model_uri: "models/model.pkl"
+
 
-# Use a section of the TLC Trip Record Dataset for model development
-# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
-INGEST_DATA_LOCATION: dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled
-# Specify the format of the dataset
-INGEST_DATA_FORMAT: spark_sql
-# Use a section of the TLC Trip Record Dataset for model development
-# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
-INGEST_SCORING_DATA_LOCATION: dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled
-# Specify the format of the dataset
-INGEST_SCORING_DATA_FORMAT: spark_sql
 # Override the default train / validation / test dataset split ratios
 SPLIT_RATIOS: [0.75, 0.125, 0.125]
-# Specify the output location of the batch scoring predict step
-SCORED_OUTPUT_DATA_LOCATION: "taxi_regression_batch_scoring"
-# Specify the output format of the batch scoring predict step
-SCORED_OUTPUT_DATA_FORMAT: table
\ No newline at end of file
+
+INGEST_STEP_CONFIG:
+    # For different options please read: https://github.com/mlflow/mlp-regression-template#ingest-step
+    using: spark_sql
+    location: "dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled"
+    sql: SELECT * FROM delta.`dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled`
```

Comment:
I don't see any configuration for the `ingest_scoring_step`. Is this intended?

Reply:
We decided to merge those configs in Register step :) to make it more consistent with other steps.